readme.md

```diff
@@ -58,6 +58,7 @@ Things that produce TAP output.
 - [qunit-tap](https://github.com/twada/qunit-tap) - TAP output for QUnit.
 - [jasmine-reporters](https://github.com/larrymyers/jasmine-reporters) - TAP output for Jasmine.
 - [karma-tap-reporter](https://github.com/fumiakiy/karma-tap-reporter) - TAP output for Karma.
+- [zora](https://github.com/lorenzofox3/zora) - ES6 test runner that works like charm without babel.
```

Comment:
The description needs to be better written.

The item should be added to the end.

Reply:
fixed